libs/npm/core/src/components/grid-table/GridTable.tsx

```diff
@@ -283,6 +283,17 @@ export const GridTable = <T extends object>({
     }
   }, [initialSortBy, instance, onChangeSort, sortBy]);
 
+  React.useEffect(() => {
+    instance.toggleAllRowsSelected(false);
```

Comment:
Can you provide comments on what is going on here?  Also how do we know we won't run into a runtime error with your hardcoded array positions?

Reply:
When selecting a different row either by mouse click or using the previous/next navigation buttons, we need to unselect the previous selected row(s) and highlight the current one. For the hardcoded array positions, it seems working fine and as expected based on the testing, however, an update has been made to address it here for a peace of mind. Thanks